core/dt.py

```diff
@@ -635,32 +384,32 @@ def dt_model():
     model = 'model = "{}";'.format(conf['model'])
     return model
 
-
-def dt_create_root_node(is_zephyr=False):
+def dt_create_root_node(cfg, model, os):
```

Comment:
Some of the functions got comment but some did not. Should we standardize that all functions to have comments?